yarn-project/pxe/src/database/database.ts

```
@@ -24,6 +24,18 @@ export interface Database extends ContractDatabase {
    */
   getAuthWitness(messageHash: Fr): Promise<Fr[]>;
 
+  /**
+   * Adding a mint to the pez dispenser.
+   * @param mint - An array of field elements representing the mint.
+   */
+  addMint(mint: Fr[]): Promise<void>;
+
+  /**
+   * Get the next mint from the pez dispenser.
+   * @returns A promise that resolves to an array of field elements representing the mint.
+   */
+  popMint(): Promise<Fr[] | undefined>;
```

do we want to throw or just return undefined?